pyaedt/modeler/cad/Primitives.py

```diff
@@ -2863,8 +2863,8 @@ def sweep_along_vector(self, objid, sweep_vector, draft_angle=0, draft_type="Rou
 
         Returns
         -------
-        bool
-            ``True`` when successful, ``False`` when failed.
+        pyaedt.modeler.cad.object3d.Object3d, list
```

Comment:
It should be:

```
:class:`pyaedt.modeler.cad.object3d.Object3d` or list of :class:`pyaedt.modeler.cad.object3d.Object3d`
            Objects created.
```

Reply:
I'll correct it

Reply:
Like this?
![image](https://github.com/ansys/pyaedt/assets/119294121/01bae0e3-ca13-4a18-8adc-91dc708e3641)